packages/@ionic/cli-utils/src/lib/project/angular/app-scripts.ts

```diff
@@ -0,0 +1,8 @@
+import { compileNodeModulesPaths, resolve } from '@ionic/cli-framework/utils/npm';
+
+export async function importAppScripts(projectDir: string): Promise<any> {
```

Comment:
Define two functions, one for importing `@angular-devkit/schematics` and one for `@angular-devkit/schematics/tools`.